Source/WebCore/rendering/RenderLayerCompositor.cpp

```diff
@@ -4829,6 +4829,9 @@ static inline ScrollCoordinationRole scrollCoordinationRoleForNodeType(Scrolling
 ScrollingNodeID RenderLayerCompositor::attachScrollingNode(RenderLayer& layer, ScrollingNodeType nodeType, ScrollingTreeState& treeState)
 {
     auto* scrollingCoordinator = this->scrollingCoordinator();
+    if (!scrollingCoordinator)
+        return 0;
```

Comment:
is 0 a valid node ID?

Reply:
No (there's a similar zero return value a few lines down).